### PR TITLE
https://issues.redhat.com/browse/ACM-8880

### PR DESCRIPTION
**Pull request by oafischer**

### CODE REVIEW

File: troubleshooting/trouble_observability.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ observabilityaddons.observability.open-cluster-management.io
 observatoria.core.observatorium.io
 ----
 
-. If you create your own storageClass for a Bare Metal cluster, see link:https://source.redhat.com/groups/public/openshiftqe/openshiftqeknowledgebase/openshift_qe_knowledge_base_wiki/how_to_create_an_nfs_provisioner_in_the_cluster_or_out_of_the_cluster[How to create an NFS provisioner in the cluster or out of the cluster]. 
+. If you create your own storageClass for a Bare Metal cluster, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html-single/storage/index#persistent-storage-using-nfs[Persistent storage using NFS].
```
Comment (swopebe) [inline troubleshooting/trouble_observability.adoc]: I think like most of the doc, we shoul d be pointing to 4.12, remember PM focus to link to 4.12 as what is known -- only Hypershift links to later.

Comment (dockerymick) [inline troubleshooting/trouble_observability.adoc]: ACM 2.9 was tested on OCP 4.12. However, we can link to OCP 4.14 from the HyperShift docs. I have to find that convo, I forget if you were on PTO when this was discussed. 

```suggestion
. If you create your own storageClass for a Bare Metal cluster, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html-single/storage/index#persistent-storage-using-nfs[Persistent storage using NFS].
```

File: troubleshooting/trouble_observability.adoc [hunk 2]
```diff
@@ -33,7 +33,7 @@ observabilityaddons.observability.open-cluster-management.io
...
 . To ensure that the observability component can find the default storageClass, update the `storageClass` parameter in the `multicluster-observability-operator` CRD. Your parameter might resemble the following value:
```
Comment (dockerymick) [inline troubleshooting/trouble_observability.adoc]: Will you also update the "CRD" instance. Change to custom resource definition
```suggestion
. To ensure that the observability component can find the default storageClass, update the `storageClass` parameter in the `multicluster-observability-operator` custom resource definition. Your parameter might resemble the following value:
```

**Review by dockerymick — APPROVED** [review]
Made some comments

**Review by swopebe — CHANGES_REQUESTED** [review]

File: troubleshooting/trouble_observability.adoc (hunk 1 above)
Comment (oafischer) [reply troubleshooting/trouble_observability.adoc]: Updated

**Review by dockerymick — APPROVED** [review]

**Review by berenss — APPROVED** [review]
lgtm

**Pull request merged**